test/unit/tasks/seeds_test.rb

```diff
@@ -205,4 +205,12 @@ def seed(*seed_files)
     view_permissions = Permission.all.select { |permission| permission.name.match(/view/) && permission.name != 'view_settings' }
     assert_equal [], view_permissions - Role.unscoped.find_by_name('Viewer').permissions
   end
+
+  test "adds description to template kind" do
+    seed('070-provisioning_templates.rb')
+    tmpl_kind = TemplateKind.unscoped.find_by_name('provision')
+    tmpl_kind.description = 'test description provision kind'
+    tmpl_kind.save!
+    assert TemplateKind.unscoped.find_by_name('provision')
+  end
```

Comment:
This test does not seem to be right, you want to make sure that the seed script 070 actually creates description. So simplify it a bit, just:

```ruby
seed('070-provisioning_templates.rb')
tmpl_kind = TemplateKind.unscoped.find_by_name('provision')
assert_equal "Expected description", tmpl_kind.description
```

Reply:
Thanks for the inputs. I have modified the test but, foreman tests seem to go red because of [https://ci.theforeman.org/job/test_develop_pr_core/19834/](https://ci.theforeman.org/job/test_develop_pr_core/19834/) . I had rebased and sent but still, this seems to be an issue. Is this being caused by my PR ?